release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -429,6 +429,17 @@ Starting in {product-title} 4.14, Extended Update Support (EUS) is extended to t
 
 In {product-title} {product-version}, the installation program uses the Cluster API instead of Terraform to provision cluster infrastructure during installations on {ibm-cloud-title}.
 
+
+[id="ocp-4-19-installation-and-update-azure-outbound-access-vms_{context}"]
+==== Outbound access for VMs in {azure-first} will be retired
+On 30 September 2025, the default outbound access connectivity for all new virtual machines (VMs) in {azure-first} will be retired. To enhance security, {azure-short} is moving towards a secure-by-default model where default outbound access to the internet will be turned off.
+
+As a result, after 30 September 2025 nodes that you deploy in {product-title} using {azure-short} will not provide for outbound access connectivity unless explicitly configured.
+
+Existing VMs and existing and new installer-provisioned infrastructure installations are not affected by the retirement. However, for user-provisioned infrastructure installations on {azure-short} you must assign a public IP address for load balancers.
```

Comment:
To be clear, nothing needs to be explicitly configured here, the installer sets up an outbound rule on the load balancer by default.

Reply:
This is better. LGTM.